paper-slider.html

```diff
@@ -483,6 +488,10 @@
         'up pageup end': '_incrementKey'
       },
 
+      attached: function() {
+        this._isRTL = window.getComputedStyle(this)['direction'] === 'rtl';
```

Comment:
The `getComputedStyle()` in `attached` is costly, and could degrade performance significantly especially if there are multiple instances of this element. Don't know of any alternatives tho - thoughts @frankiefu?